controllers/snapshot/snapshot_adapter.go

```diff
@@ -187,6 +191,10 @@ func (a *Adapter) EnsureApplicationSnapshotEnvironmentBindingExist() (results.Op
 		return results.ContinueProcessing()
 	}
 
+	if !gitops.IsSnapshotCreatedByPushEvent(a.snapshot) {
+		return results.ContinueProcessing()
```

Comment:
tiny nit: maybe also add an a.logger.Info call informing that the snapshot won't be released because it was not created for a push event.

Reply:
Added. Thanks